### PR TITLE
WD-9843 Rebrand server s390x

### PR DESCRIPTION
**Pull request by carkod**
## QA

- Check https://ubuntu-com-13671.demos.haus/download/server/s390x against [copy docs](https://docs.google.com/document/d/1d85IfqqxZYYZ0orj5lBhqLWLGpLYWRLdvs75lYsplGs/edit)
    - Be sure to test on mobile, tablet and desktop screen sizes

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-9483

## Screenshots (design review)

s390x

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# syntax=docker/dockerfile:experimental
```
Comment (carkod) [inline Dockerfile:1]: This was added to fix demos, but it looks like they are working now without this change.

**Comment by juanruitina** [comment]
Only one thing: the copy under Installation guides doesn't match the copy doc. Please note it's different for /download/server/arm and the other pages, I understand that affects partials.

**Comment by carkod** [comment]
> Only one thing: the copy under Installation guides doesn't match the copy doc. Please note it's different for /download/server/arm and the other pages, I understand that affects partials.

@juanruitina Sorry, should have checked, updated!

**Comment by juanruitina** [comment]
UX+1!

File: templates/download/server/s390x.html [hunk 2]
```diff
@@ -3,148 +3,185 @@
...
-</div>
-<div class="p-strip">
-  <div class="row">
-    <div class="col-6">
-      <h2>Ubuntu Server</h2>
+<div class="p-strip is-shallow">
+  <div class="row--50-50">
+    <div class="col">
+      <div class="p-section">
+        <h1>Ubuntu Server LTS <br class="u-hide--small" /> IBM Z and LinuxONE</h1>
```
Comment (petesfrench) [inline templates/download/server/s390x.html:13]:
```suggestion
        <h1>Ubuntu Server LTS for <br class="u-hide--small" /> IBM Z and LinuxONE</h1>
```
As in the doc

**Comment by petesfrench** [comment]
Rogue dot under the phone input:
image

File: templates/download/server/s390x.html [hunk 3]
```diff
@@ -3,148 +3,185 @@
...
+            <option value="z15 T02/LinuxONE III LT2">z15 T02/LinuxONE III LT2</option>
+            <option value="z15/T02/LinuxONE III LY2 - 1/2 drawer (up to 65 cores)"> - 1/2 drawer (up to 65 cores)</option>
+
+            <option value="z15 T01/LinuxONE III LT1">z15 T01/LinuxONE III LT1 </option>
+            <option value="z15/LinuxONE III - 1 drawer (up to 34 cores)"> - 1 drawer (up to 34 cores)</option>
+            <option value="z15/LinuxONE III - 2 drawers (up to 71 cores)"> - 2 drawers (up to 71 cores)</option>
+            <option value="z15/LinuxONE III - 3 drawers (up to 108 cores)"> - 3 drawers (up to 108 cores)</option>
+            <option value="z15/LinuxONE III - 4 drawers (up to 190 cores)"> - 4 drawers (up to 190 cores)</option>
+
+            <option value="z16 A01">z16 A01</option>
```
Comment (petesfrench) [inline templates/download/server/s390x.html:83]:
```suggestion
            <option value="z16 A01" disabled>z16 A01</option>
```

**Review by petesfrench — COMMENTED** [review]
A couple comments on the code.

Also, the thank you page says 'thank you for downloading' before you have downloaded which is a little confusing

File: Dockerfile (hunk 1 above)
Comment (carkod) [reply Dockerfile:1]: No, we are adding it back, the fix was removing it.

File: templates/download/server/s390x.html [hunk 4]
```diff
@@ -3,148 +3,185 @@
...
+            <option value="z14 ZR1/Rockhopper II" disabled> z14 ZR1/Rockhopper II</option>
+            <option value="z14 ZR1/Rockhopper II - 1/2 drawer (up to 30 cores)"> - 1/2 drawer (up to 30 cores)</option>
+
+            <option value="z14/Emperor II" disabled>z14/Emperor II</option>
+            <option value="z14/Emperor II - 1 drawer (up to 33 cores)"> - 1 drawer (up to 33 cores)</option>
+            <option value="z14/Emperor II - 2 drawers (up to 69 cores)"> - 2 drawers (up to 69 cores)</option>
+            <option value="z14/Emperor II - 3 drawers (up to 105 cores)"> - 3 drawers (up to 105 cores)</option>
+            <option value="z14/Emperor II - 4 drawer (up to 170 cores)"> - 4 drawer (up to 170 cores)</option>
+
+            <option value="z15 T02/LinuxONE III LT2">z15 T02/LinuxONE III LT2</option>
```
[the remark this answers is not in the document]
Comment (carkod) [reply templates/download/server/s390x.html:74]: Why disable it? The copy doc is adding this?

File: templates/download/server/s390x.html [hunk 5]
```diff
@@ -3,148 +3,185 @@
...
+          <p><a href="/server/docs">Ubuntu Server documentation&nbsp;&rsaquo;</a></p>
+        </div>
+      </div>
+      <hr class="p-rule" />
+      <div class="row">
+        <div class="col-3 col-medium-2">
+          <h3 class="p-heading--5">Helping hands</h3>
+        </div>
+        <div class="col-6 col-medium-3">
+          <p>If you get stuck, help is always at hand.</p>
```
[the remark this answers is not in the document]
Comment (carkod) [reply templates/download/server/s390x.html]: Yeah this is the problem that stops us from creating a partial it's different in every page. Removed.

File: templates/download/server/s390x.html (hunk 4 above)
[the remark this answers is not in the document]
Comment (petesfrench) [reply templates/download/server/s390x.html]: I thought it was one of the visual headings and therefore shouldn't be enabled?

**Review by petesfrench — APPROVED** [review]
LGTM - form seems to go through on client side and no errors in sentry, but couldn't find it in Mrketo to check

**Pull request merged**